python/cac_tripplanner/cac_tripplanner/views.py

```diff
@@ -3,5 +3,8 @@
 def home(request):
     return render_to_response('home.html')
 
+def map(request):
```

Comment:
Sounds fine to me, I default to class based because that's what we have been doing on other projects and what I started on; but, I don't see a lot of inheritance/re-use going on in this project so we can probably just use functions.